README.md

```diff
@@ -95,6 +95,7 @@ const netlifyIdentity = require("netlify-identity-widget");
 
 netlifyIdentity.init({
   container: "#netlify-modal" // defaults to document.body,
+  APIUrl: "https://some-identity-instance.example.com" // Set an absolute URL for the identity endpoint
```

Comment:
I'll add a note to avoid it, and where you might need it (when serving from a domain that differs from where the endpoint lives, e.g for apps served fro files like cordova).